### ISSUE TITLE
Partial timestamp and missing event ID in log line

### ISSUE THREAD

**Issue by sgpinkus** [linux-audit/audit-userspace#262]
I'm using auditd=1:2.8.4-3 on Debian. I got an event in my audit.log with an incomplete timestamp and no event ID in the first line of the event:

```
type=SYSCALL msg=audit(16523210---): arch=c000003e syscall=87 success=yes exit=0 a0=7f867d66a3ed a1=7f867d66a3ed a2=0 a3=792f18 items=2 ppid=2275 pid=16746 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=(none) ses=2 comm=4C5320546872656164 exe="/usr/lib/firefox-esr/firefox-esr" subj==unconfined key="delete"
type=CWD msg=audit(1652321038.100:23444): cwd="/home/sam"
type=PATH msg=audit(1652321038.100:23444): item=0 name="/home/sam/.mozilla/firefox/baey2He4.default/" inode=15861713 dev=fe:01 mode=040700 ouid=1000 ogid=1000 rdev=00:00 nametype=PARENT cap_fp=0 cap_fi=0 cap_fe=0 cap_fver=0 cap_frootid=0
type=PATH msg=audit(1652321038.100:23444): item=1 name="/home/sam/.mozilla/firefox/baey2He4.default/webappsstore.sqlite-wal" inode=15860647 dev=fe:01 mode=0100644 ouid=1000 ogid=1000 rdev=00:00 nametype=DELETE cap_fp=0 cap_fi=0 cap_fe=0 cap_fver=0 cap_frootid=0
type=PROCTITLE msg=audit(1652321038.100:23444): proctitle="/usr/lib/firefox-esr/firefox-esr"
```

Surprisingly "16523210---" is a valid UNIX timestamp, at least according to `date --date="@16523210---`, so I guess there may be a legit reason auditd can't provide a full timestamp. The real issue is the event ID is missing.

**Comment by stevegrubb** [comment]
I have never seen such an event. There is no code with exactly 3 dashes in user space or the kernel's audit code. This is created by the kernel here:  https://github.com/linux-audit/audit-kernel/blob/main/kernel/audit.c#L1910

**Comment by sgpinkus** [comment]
OK good to know - I think :/.

I'll dig into what happened when I get a chance and let you know if I figure anything out.

**Comment by sgpinkus** [comment]
Having reviewed the code I can't see anyway you could end up with the "---". I looked at version that (I think) was used in the v5.10 kernel that was being used. @stevegrubb any ideas?

**Comment by stevegrubb** [comment]
As I mentioned, there is no code in the kernel's audit or user space code with 3 dashes. I have no idea how a conversion of a %u format specifier becomes non-numeric. I think this can be closed as there is nothing that I see as actionable.

**Comment by stevegrubb** [comment]
Closing this. There is nothing I can think of that caused this.

**Comment by sgpinkus** [comment]
But it happened. The reason I noticed is I had a script tailing the log a parsing the line and it threw an error because it could not parse the given line. I can't see how it would happen either. Only thing I could think of was A. it was something from an older version or a Debian patch B. some kind of weird OOM condition caused it as well as the same thing causing the time not to be reported fully. Anyway I did a ~hour long code review and couldn't solve. Agree to close ... will raise again if see again.
